tools/testbuild.sh

```diff
@@ -244,11 +244,11 @@ function configure {
     varname=`echo $setting | cut -d'=' -f1`
     if [ ! -z "$varname" ]; then
       echo "  Disabling $varname"
-      sed -i -e "/$varname/d" $nuttx/.config
```

Comment:
kconfig tweak would be better here as well

Reply:
done